app/jobs/document_proofing_job.rb

```diff
@@ -57,9 +50,9 @@ def perform(
       doc_auth_client.post_images(
         front_image: front_image,
         back_image: back_image,
-        selfie_image: selfie_image || '',
+        selfie_image: nil,
```

Comment:
do we want to do the work to dig down into the clients and remove `selfie_image` as an argument too? or future PR?

Reply:
that is going to be a future PR